easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2020a.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'CD-HIT'
+version = '4.8.1'
+
+homepage = 'http://weizhong-lab.ucsd.edu/cd-hit/'
+description = """ CD-HIT is a very widely used program for clustering and 
+ comparing protein or nucleotide sequences."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/weizhongli/cdhit/releases/download/V%(version)s/']
+sources = ['%(namelower)s-v%(version)s-2019-0228.tar.gz']
+checksums = ['26172dba3040d1ae5c73ff0ac6c3be8c8e60cc49fc7379e434cdf9cb1e7415de']
+
+dependencies = [('Perl', '5.30.2')]
```

According to the documentation CD-HIT also needs `zlib`
```suggestion
dependencies = [
    ('Perl', '5.30.2'),
    ('zlib', '1.2.11'),
]
```